child_compassion/wizards/child_hold_wizard.py

```diff
@@ -35,7 +35,9 @@ class ChildHoldWizard(models.TransientModel):
         ('Sponsored', _('Sponsored')),
         ('Sub Child Hold', _('Sub Child Hold'))], required=True)
     hold_expiration_date = fields.Datetime(required=True)
-    primary_owner = fields.Char(required=True)
+    primary_owner = fields.Char(required=True,
+                                default=lambda self:
+                                self.default_primary_owner())
```

Comment:
You could directly put `lambda self: self.env.user.name` instead of doing a function for this.